test/integration_test.js

```diff
@@ -542,6 +546,62 @@ describe("Integration tests", function() {
           });
         });
 
+        describe("CLIENT_WINS strategy with transformers", () => {
+          beforeEach(() => {
+            return collectionTestSync(tasksTransformed, testData, {
+              strategy: Kinto.syncStrategy.CLIENT_WINS
+            }).then(res => syncResult = res);
+          });
+
+          it("should publish resolved conflict using local version", () => {
+            expect(syncResult.published).to.have.length.of(1);
+            expect(recordsEqual(syncResult.published[0], {
+              id: conflictingId,
+              title: "task4-local",
+              done: false,
+            })).eql(true);
+          });
```

Comment:
The code path for changes that come from `pullChanges` is pretty different, and records are decoded before they are checked for conflicts, so these tests doesn't contribute anything in the current codebase. However, the functionality is kind of important to me. I'd be OK with deleting them, or keeping them.